tests/load/gcp/lma_startup_script.tpl.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+set -euxo pipefail
+
+systemctl disable systemd-tmpfiles-clean.timer --now
+microk8s status --wait-ready
+microk8s.enable storage dns ingress
+
+IPADDR=$(ip -4 route | grep default | grep -oP 'src \K\S+')
+
+# Occasionally, enabling metallb fails with the following error:
+# The connection to the server 127.0.0.1:16443 was refused - did you specify the right host or port?
+# Try enabling it a loop:
+until microk8s.enable metallb:$IPADDR-$IPADDR; do sleep 5; done
```

Comment:
This feels very much like a hack, have you engaged the microk8s team to find out what's going on here?